src/openforms/conf/base.py

```diff
@@ -47,6 +47,16 @@
     ("nl", _("Dutch")),
 ]
 
+LANGUAGE_COOKIE_HTTPONLY = True
+
+LANGUAGE_COOKIE_NAME = "openforms_language"
+
+LANGUAGE_COOKIE_SAMESITE = config(
+    "LANGUAGE_COOKIE_SAMESITE", default="None" if IS_HTTPS else "Lax"
+)
+
+LANGUAGE_COOKIE_SECURE = IS_HTTPS
```

Comment:
final remark - can you remove the blank lines between the language settings to be consistent with the rest of the settings file? :grimacing: